codegen/src/main/java/com/paypal/selion/reader/YamlV1Reader.java

```diff
@@ -77,16 +77,14 @@ public void processPage(FileSystemResource resource) throws IOException {
                     setBaseClassName((String) map.get("Value"));
                 }
 
-                if ("platform".equals(map.get(KEY))) {
-                    if (!platformDefined) {
-                        TestPlatform currentPlatform = TestPlatform.identifyPlatform((String) map.get("Value"));
-                        if (currentPlatform == null) {
-                            String dataFile = new File(fileName).getAbsolutePath();
-                            throw new IllegalArgumentException("Missing or incorrect platform in Data file:" + dataFile);
-                        }
-                        setPlatform(currentPlatform);
-                        platformDefined = true;
+                if ("platform".equals(map.get(KEY)) && !platformDefined) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Avoid using Literals in Conditional Statements](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1433391601/issues/source?bid=2928425&fileBranchId=2989551#l80)